app-webpack/templates/ssr/ts/middlewares/render.ts

```diff
@@ -0,0 +1,54 @@
+import { ssrMiddleware } from 'quasar/wrappers';
+
+// This middleware should execute as last one
+// since it captures everything and tries to
+// render the page with Vue
+
+export default ssrMiddleware(({ app, resolve, render, serve }) => {
+  // we capture any other Express route and hand it
+  // over to Vue and Vue Router to render our page
+  app.get(resolve.urlPath('*'), (req, res) => {
+    res.setHeader('Content-Type', 'text/html');
+
+    render(/* the ssrContext: */ { req, res })
+      .then(html => {
+        // now let's send the rendered html to the client
+        res.send(html);
+      })
+      .catch(err => {
```

Comment:
Good catch! Will do.

ESLint and strict TS would normally go crazy about this, but when working on the Electron TS template, I discovered that since these files are not included in the root tsconfig, they are not really "acting as usual". I think we should either directly include them into the root tsconfig's `include` section or create tsconfigs for each folder, then connect them to the root using `references`. Otherwise, we'll not get the full benefits of TS detection and strict linting.
Here is an example I found while working on Electron TS template:
```ts
// The root level of electron-main.ts
let mainWindow: BrowserWindow | undefined;
// mainWindow: BrowserWindow
mainWindow.loadURL() // will end up with "can't access `loadURL` of undefined" at runtime
```
So, IMO that is unacceptable and we should address that. If you have any preferred solution among those two or have another solution, please let me know.